src/microbe_stage/MicrobeStage.cs

```diff
@@ -496,6 +496,9 @@ public void OnReturnFromEditor()
 
         StartMusic();
 
+        // Apply language settings (to fix bug, see #1885)
```

Comment:
This comment should have the full explanation that due to the stage scene tree being unattached during editor, if language was changed while in the editor, it doesn't properly propagate. So language is reapplied here to make sure that the stage doesn't continue to use the wrong language.